h/views/api_auth.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h.auth.services import TOKEN_TTL
+from h.exceptions import OAuthTokenError
+from h.util.view import json_view
+
+
+@json_view(route_name='token', request_method='POST')
```

Comment:
My main opinion is to be consistent. It will make our endpoints more predictable and less prone to needless cycles on small differences like encoding. That said, if we expect this to be used by other orgs, we should support JSON bodies as well. I have no data on this but most posting libraries are not `application/x-www-form-urlencoded` by default - they tend to be JSON body by default. Adding a layer of abstraction before interacting with inputs should handle this in a way that is reusable for any API we expect to open to the public.